backend/pyproject.toml

```diff
@@ -40,9 +39,11 @@ unstructured = {extras = ["doc", "docx"], version = "^0.12.5"}
 pgvector = "^0.2.5"
 psycopg2-binary = "^2.9.9"
 asyncpg = "^0.29.0"
-langchain-core = "^0.1.44"
+langchain-core = "^0.1.50"
 pyjwt = {extras = ["crypto"], version = "^2.8.0"}
 langchain-anthropic = "^0.1.8"
+langgraph-sdk = "^0.1.1"
+langgraph-cli = "^0.1.3"
```

Comment:
The packages https://pypi.org/project/langgraph-sdk/ and https://pypi.org/project/langgraph-cli/ are missing metadata (the license information etc.).

Reply:
updated